pkg/webhook/cspc.go

```diff
@@ -103,9 +105,12 @@ func (wh *webhook) validateCSPC(ar *v1beta1.AdmissionReview) *v1beta1.AdmissionR
 	} else if req.Operation == v1beta1.Create {
 		klog.V(5).Infof("Admission webhook create request for type %s", req.Kind.Kind)
 		return wh.validateCSPCCreateRequest(req)
+	} else if req.Operation == v1beta1.Delete {
+		klog.V(5).Infof("Admission webhook delete request for type %s", req.Kind.Kind)
+		return wh.validateCSPCDeleteRequest(req)
 	}
 
-	klog.V(2).Info("Admission wehbook for PVC not " +
+	klog.V(2).Info("Admission wehbook for CSPC not " +
```

Comment:
its PVC

Reply:
change this to V(5) ?

Reply:
The function name is validateCSPC so it should be CSPC. correct ?
```golang
      klog.V(5).Info("Admission wehbook for CSPC not " +
         "configured for operations other than CREATE, UPDATE and DELETE")
```

Reply:
ahh, didn't check name, then remove this log.